src/index.ts

```diff
@@ -197,6 +197,7 @@ export type {
   ResumeToken,
   UpdateDescription
 } from './change_stream';
+export { AuthContext, AuthContextOptions } from './cmap/auth/auth_provider';
```

Comment:
```suggestion
export type { AuthContext, AuthContextOptions } from './cmap/auth/auth_provider';
```

this will also remove the need to add these to the `index.test.ts` file.

Reply:
Updated.

Reply:
the new fields in `index.test.ts` should be reverted now, they should be going red